subsys/net/ip/rpl.c

```diff
@@ -2044,7 +2044,7 @@ static bool net_rpl_process_parent_event(struct net_if *iface,
 	return ret;
 }
 
-static bool net_rpl_repair_root(u8_t instance_id)
+bool net_rpl_repair_root(u8_t instance_id)
```

Comment:
Is this worth exposing, why would it be needed to be used from shell? (besides some low level rpl testing maybe). Repairing the rpl network is meant to be automatic, and afaik, rpl is designed to be smart enough to do so relevantly.